portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+import java.time.LocalDateTime;
+
+/** Represents a comment with related details. */
+public final class Comment {
+    private final String name;
```

Comment:
Nit: for clarity, maybe "commenterName" or "commenter" ? it's not clear what "name" it is at the moment.

Reply:
Fixed